autoload/handlers.vim

```diff
@@ -446,6 +446,13 @@ def s:processReferencesReply(lspserver: dict<any>, req: dict<any>, reply: dict<a
 			text: text})
   endfor
 
+  if opt.lspOptions.referencesCloseOnSelect
+    augroup LspReferences
+      autocmd!
+      autocmd BufLeave * lclose
```

Comment:
As this autocmd applies to all the buffers, this may lead to unexpected side-effects.
Can you change this autocmd to use the <buffer> tag so that it only applies to the
current buffer?

Reply:
It'll affect all location list from other plugins as well, most likely. :/
I'll revert this one. I think it should go to vimrc of users, because it's not how locationlist is supposed to behave.